include/boost/gil/image_processing/filter.hpp

```diff
@@ -135,6 +137,64 @@ void median_filter(SrcView const& src_view, DstView const& dst_view, std::size_t
     }
 }
 
+namespace detail {
+
+template<typename KernelT>
+void getGaussianKernel(KernelT& kernel,
+                       long int kernel_size,
+                       double sigma,
+                       bool normalize = true)
+{
+    if (!(kernel_size & 0x1))
```

Comment:
Some may read it as fancy syntax. The old school `size % 2 != 0` will generate the very same assembly :)